### DISCUSSION TITLE
Can we use 2 Formik Controls with the same name inside Formik?

### DISCUSSION THREAD

**Discussion by akk6** [jaredpalmer/formik#3077] [Q&A]
I have a requirement where I have to conditionally display only one Form.Control but the values entered in them should be updated in the same state variable in my React Component

**Control 1:**
`<Form.Control name="expiryFormdate" as="select"...>`

**Control 2:**
`<Field name="expiryFormdate" component={this.datePickerComponent}>`

At a time only one Component will be visible and another one will be conditionally hidden. Would using the same name "expiryFormdate" for both of them cause any issues even if only one is visible?

**Comment by johnrom** [comment]
You'll run into one issue technically -- `validate` will only be passed by the first one. If you're using Field-level validations this might be an issue. You cannot validate them two different ways.

Otherwise, I think you're OK.
